cylc/flow/scripts/cylc_help.py

```diff
@@ -238,7 +238,7 @@ def category_help(category):
 control_commands['kill'] = ['kill']
 control_commands['hold'] = ['hold']
 control_commands['release'] = ['release', 'unhold']
-control_commands['spawn'] = ['spawn']
+control_commands['set-outputs'] = ['set-outpus']
```

Comment:
Doh, spelling error!  I'll push up a hot fix, not worth a PR.

Reply:
(Found post merge).

Reply:
Opps

Reply:
🍌